src/core/Workbench.tsx

```diff
@@ -1,342 +1,45 @@
-    dragging: false,
-  });
-  const el = useCallback((node) => {
-    if (node !== null) {
-      dispatch({ type: "widthknown", width: node.offsetWidth });
-    }
-  }, []);
-  return [handleState, el, dispatch];
-}
-
-function useHeightDraggableHandle(id: string, invert: boolean) {
-  const [handleState, dispatch] = useReducer(handleHeightDragging, {
-    id,
-    invert,
-    dragging: false,
-  });
-  const el = useCallback((node) => {
-    if (node !== null) {
-      dispatch({ type: "heightknown", height: node.offsetHeight });
-    }
-  }, []);
-  return [handleState, el, dispatch];
-}
-
-enum CssColor {
-  darkgray = "darkgray",
-  dimgray = "dimgray",
-  slategray = "slategray",
-}
-
-interface TextIconProps {
-  name: string;
-  color?: CssColor;
-}
-
-function TextIcon({ name, color }: TextIconProps) {
-  return (
-    <span style={{ color }} className="material-icons icon">
-      {name}
-    </span>
-  );
-}
-
-function Tool({ name, close, children }) {
-  const [minimizeChosen, _, setMinimize] = useToggle(false);
-  useMouseUpAnywhere(setMinimize(false));
-  const chosenClass = "minimize " + (minimizeChosen ? "chosen" : "");
-  return (
-    <>
-      <header className="tool-header">
-        <ul className="tool-header-main">
-          <li>{name}</li>
-        </ul>
-        <ul className="tool-header-controls">
-          <li
-            className={chosenClass}
-            onClick={close}
-            onMouseDown={setMinimize(true)}
-            onMouseUp={setMinimize(false)}
-          >
-            <TextIcon color={CssColor.dimgray} name="minimize" />
-          </li>
-        </ul>
-      </header>
-      <section className="tool">{children}</section>
-    </>
-  );
-}
-function BrowseSidebar({ navEl, width, dispatchLeft }) {
-  const [open, toggleOpen] = useToggle(true);
-  return open ? (
-    <>
-      <nav ref={navEl} style={{ width: width + "px" }}>
-        <Tool name="Browse" close={toggleOpen}>
-          <p>This is the browse tool content</p>
-        </Tool>
-      </nav>
-      <div
-        id="lefthandle"
-        className="handle"
-        onMouseDown={dispatchLeft}
-        onMouseMove={dispatchLeft}
-      />
-    </>
-  ) : (
-    <>
-      <aside className="left collapsed">
-        <ul>
-          <li className="left" onClick={toggleOpen}>
-            <TextIcon name="manage_search" />
-            <span className="left">Browse</span>
-          </li>
-        </ul>
-      </aside>
-      <div id="lefthandle" className="handle collapsed" />
-    </>
-  );
-}
-
-function Tools({ toolsEl, width, dispatchRight }) {
-  const [open, toggleOpen] = useToggle(true);
-  return open ? (
-    <>
-      <div
-        id="righthandle"
-        className="handle"
-        onMouseDown={dispatchRight}
-        onMouseMove={dispatchRight}
-      />
-      <div ref={toolsEl} id="tools" style={{ width: width + "px" }}>
-        <Tool name="History" close={toggleOpen}>
-          <p>This is the history tool content</p>
```

Comment:
Did we remove the history panel?

Reply:
According to JR's mocks, this will be a separate page (Recent) which was added to the left pannel